### ISSUE TITLE
BUG: Defintion of Einstandsgemeinschaften

### ISSUE THREAD

**Issue by MImmesberger** [iza-institute-of-labor-economics/gettsim#738]
Sorry to revive this discussion again: 

### Status Quo

In our current implementation, Einstandsgemeinschaft refers to couples who live together (specified via `p_id_einstandspartner`). This follows the definition in SGB II: [§7 Abs. 3a SGB II](https://www.gesetze-im-internet.de/sgb_2/__7.html) defines a Einstandsgemeinschaft as two people living together with a mutual willingness to take care for each other (so no children included here).

Here, the Einstandsgemeinschaft is part of the Bedarfsgemeinschaft. It seems like the purpose of the Bedarfsgemeinschaft in SGB II is to summarize couples who are not necessarily married under one term (afaik there is no connection to transfers beyond the membership in a Bedarfsgemeinschaft).

In SGB XII, the Einstandsgemeinschaft is the group that is entitled for some transfer and **includes underage children** who live in the same household and cannot cover their own needs. See, for example, this definition of [BMAS](https://www.bmas.de/DE/Soziales/Sozialhilfe/Leistungen-der-Sozialhilfe/leistungen-der-sozialhilfe.html).

>Bei in Privathaushalten lebenden Personen werden zusammenwohnende Partner sowie im Haushalt lebende minderjährige Kinder als sog. Einstandsgemeinschaft betrachtet. Eine Einstandsgemeinschaft ist dadurch gekennzeichnet, dass die Partner dieser Einstandsgemeinschaft füreinander mit ihrem Einkommen und Vermögen einstehen.

### Implications for GETTSIM

It seems to me that all occurrences of `_eg` in the code should actually refer to the SGB XII term, not the SGB II term (because they are used for SGB XII transfers). Hence, we would need to adjust the creation of `eg_id` accordingly.

I don't know yet how Einstandsgemeinschaft (SGB XII) relates to Bedarfsgemeinschaft (SGB II) other than the fact that in the latter, there needs to be at least one person that is able to work.

**Comment by hmgaudecker** [comment]
Not sure. Seems to me like this is a definitional thing and we can just keep it as is -- we need that definition from SGB II in order to create `bg_id`s, right?

> I don't know yet how Einstandsgemeinschaft (SGB XII) relates to Bedarfsgemeinschaft (SGB II) other than the fact that in the latter, there needs to be at least one person that is able to work.

Sounds like it is fine to run with the `bg_id` for SGB XII as well.

**Comment by hmgaudecker** [comment]
~~After #805, there will be a very natural way to distinguish between the two!~~ Edit MI: Not true anymore.

**Comment by MImmesberger** [comment]
Superseded by #1147
